docs/system-admin-guide/authentication/oauth-applications/README.md

```diff
@@ -8,34 +8,35 @@ keywords: OAuth application settings
 ---
 # OAuth applications
 
-To activate and configure oauth applications, navigate to -> *Administration* -> *Authentication* and choose -> *Oauth applications*.
+To activate and configure OAuth applications, navigate to -> *Administration* -> *Authentication* -> *OAuth applications*.
 
-## Add a new authentication application for oauth
+## Add a new authentication application for OAuth
 
-To add a new oauth application, click the green **+ Add** button.
+To add a new OAuth application, click the green **+ Add** button.
 
-![Sys-admin-authenticatoin-oauth-applications](Sys-admin-authenticatoin-oauth-applications.png)
+![Sys-admin-authenticatoin-OAuth-applications](Sys-admin-authenticatoin-OAuth-applications.png)
 
 
 
-You can configure the following options to add your oauth application.
+You can configure the following options to add your OAuth application.
 
-1. Enter the **name** of your oauth application.
+1. Enter the **name** of your OAuth application.
 2. **Define redirect URLs** where authorized users can be redirected to.
-3. Check if the application will be used **confidentially**.
-4. (Optional) Choose **client credential flows** and define a user on whose behalf requests will be performed.
-5. Press the blue **Create** button to add your oauth application.
+3. .................................
```

Comment:
Here my 2 cents:

Set the **scope** that the OAuth application will have access to. Chose the API you want to grant access to. Multiple selections are possible. If no scope is checked, per default **api_v3** is assumed.